src/main/java/org/apache/commons/mail/util/MimeMessageParser.java

```diff
@@ -425,7 +423,7 @@ private byte[] getContent(final InputStream is)
         final BufferedInputStream isReader = new BufferedInputStream(is);
         try (BufferedOutputStream osWriter = new BufferedOutputStream(os)) {
             int ch;
-            while ((ch = isReader.read()) != -1)
+            while ((ch = isReader.read()) != 1)
```

Comment:
-1: Nothing in this PR makes sense.

Reply:
Sorry, this PR was meant to merge to the forked repo master branch, instead of this repo. Closed it and sorry again.